universal_silabs_flasher/const.py

```diff
@@ -0,0 +1,38 @@
+import enum
+
+
+class FirmwareImageType(enum.Enum):
+    # EmberZNet Zigbee firmware
+    NCP_UART_HW = "ncp-uart-hw"
+
+    # Multi-PAN RCP Multiprotocol (via zigbeed)
+    RCP_UART_802154 = "rcp-uart-802154"
+
+    # Zigbee NCP + OpenThread RCP
+    ZIGBEE_NCP_RCP_UART_802154 = "zigbee-ncp-rcp-uart-802154"
+
+    # OpenThread RCP
+    OT_RCP = "ot-rcp"
+
+
+class ApplicationType(enum.Enum):
+    GECKO_BOOTLOADER = "bootloader"
+    CPC = "cpc"
+    EZSP = "ezsp"
+    SPINEL = "spinel"
+
+
+FW_IMAGE_TYPE_TO_APPLICATION_TYPE = {
+    FirmwareImageType.NCP_UART_HW: ApplicationType.EZSP,
+    FirmwareImageType.RCP_UART_802154: ApplicationType.CPC,
+    FirmwareImageType.ZIGBEE_NCP_RCP_UART_802154: ApplicationType.CPC,
+    FirmwareImageType.OT_RCP: ApplicationType.SPINEL,
+}
+
+
+DEFAULT_BAUDRATES = {
+    ApplicationType.GECKO_BOOTLOADER: [115200],
+    ApplicationType.CPC: [460800, 115200, 230400],
+    ApplicationType.EZSP: [115200],
+    ApplicationType.SPINEL: [460800],
```

Comment:
If having one not so good RCP connected to USB or comport on the machine that using spinel protocol it can being good also having support for lower comport baudrate but is making the probing taker longer time and its not so good.

Reply:
You can override all if the probing methods with command line flags, like `--cpc-baudrate 115200,230400`. These are just the defaults.

Reply:
Then i think its the right way for more advanced user and not adding more probings that making it slower. 👍